shared/router-v2/tab-bar/container.desktop.tsx

```diff
@@ -28,10 +29,12 @@ export default Container.connect(
     _badgeNumbers: state.notifications.navBadges,
     _fullnames: state.users.infoMap,
     _justSignedUpEmail: state.signup.justSignedUpEmail,
+    _kbfsDaemonStatus: state.fs.kbfsDaemonStatus,
     _settingsEmailBanner: state.settings.email.addedEmail,
+    _tlfs: state.fs.tlfs,
+    _uploads: state.fs.uploads,
```

Comment:
this container is gonna be constantly rerunning whenever KBFS is doing anything. Is that going to cause performance issues or will it be fine since it's not calculating very much?

Reply:
I think it's fine -- `state.fs.tlfs` should rarely change, given a properly written reducer. `state.fs.uploads` usually changes every 4s when there's stuff in the journal.

Reply:
Also we have more containers doing this inside the Fs tab, making this one seem pretty small.